css/properties/column-count.json

```diff
@@ -0,0 +1,131 @@
+{
+  "css": {
+    "properties": {
+      "column-count": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/column-count",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": [
+              {
+                "version_added": "50"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "chrome_android": {
+              "version_added": "50"
+            },
+            "edge": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "52"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "1.5"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "52"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "1.5"
```

Comment:
This should be "1", there wasn't a 1.5 for Android, and 1.0 was done more or less when 3.0 went out on Desktop. See column-width entry.

Reply:
Fixed with 5341b93.